### PR TITLE
Changes from 4/2 testing

### PR DESCRIPTION
**Pull request by jamesdooley4**

### CODE REVIEW

File: src/main/deploy/deploy
Comment (jamesdooley4) [inline src/main/deploy/deploy]: ? Mistaken deleted line?

Comment (KangarooKoala) [reply src/main/deploy/deploy]: Probably? It was done in 66d7809 (pivot offsets), which wouldn't have a reason to delete that line (especially since it left the other lines in the file).

**Comment by jamesdooley4** [comment]
LGTM. I can't approve since I created the PR, so waiting for an approval before merging.

File: src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java
Comment (TAKBS2412) [inline src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java:300]: For after DCMP (since we don't have the sensors on the robot right now): this should be checking `intakeLeftSensor.isPressed()` (same for the right sensor method below).

**Review by TAKBS2412 — APPROVED** [review]

**Pull request merged**
